### DISCUSSION TITLE
Is batching prompts possible in Kobold?

### DISCUSSION THREAD

**Discussion by alexchatwin** [LostRuins/koboldcpp#739] [Q&A] [answered]
(by that I mean - can I sumit multiple prompts at once, via the API, and get a benefit to processing time)

**Comment by LostRuins** [comment, marked as answer]
No, but the API has a multiuser mode which can queue multiple requests and process them in sequence.

**Reply by alexchatwin** [reply]
Thanks for the quick reply. And for the excellent software- I'll give that a try 👍
